controllers/todos.controller.js

```diff
@@ -40,4 +40,38 @@ exports.delete = (req, res) => {
     ans.answer = "Invalid argument";
     res.status("422").send(ans);
   }
+}
+
+exports.update = (req, res) => {
+  const {id} = req.params;
+  const {body} = req;
+  const ans = {};
+
+
+    Todos.findByPk(id).then(found => {
```

Comment:
you pull the database 3 times to update 1 record. This is not right. Write a universal query that will update the record data.